### PR TITLE
Create conventions document

### LINKED ISSUE #93: Create a general styleguide
We have many unwritten convetions, and it'd be best to document them all!

We need a document, say `docs/conventions.md` that lists these unwritten convetions. Things that a linter _can't_ catch. For example, do we write it `URL` or `Url` in an identifier? How do we name our CSS selectors? How do we order rules in a CSS ruleset? When do we use arrow functions vs. function declarations? etc.

### PR DESCRIPTION
**Pull request by eddieantonio**
Fixes #93

### CODE REVIEW

File: docs/conventions.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+When it appears in the middle of an identifier, **only capitalize the first letter**:
+
+```javascript
+// good
+function fetchJson() {
+}
+// bad
+function fetchJSON() {
+}
```
Comment (eddieantonio) [inline docs/conventions.md]: Looking at this convention, I actually quite dislike it! I'd like to change the recommendation to the OPPOSITE of this; that is, we always capitalize acronyms and initialisms, including:

```javascript
/* good */
function XMLHTTPRequest() {
}
/* bad */
function XmlHttpRequest() {
}
```

I just like the way the caps look better ¯\\\_(ツ)\_/¯

Comment (jienius) [reply docs/conventions.md]: I like this

**Review by jienius — APPROVED** [review]
LGTM

**Comment by eddieantonio** [comment]
In our slack, we have a 3-1 vote on `fetchJSON()` vs `fetchJson()` (1 abstained). So I reversed the decision in the doc! Merging...

**Pull request merged**